specs/unit tests/app.test.js

```diff
@@ -0,0 +1,17 @@
+//This file tests the functionality of App specific functions
+const App = require('../../client/src/components/App.jsx');
```

Comment:
Is this working?

Reply:
Yes, it does successfully link to the App component which is then used by react-test-renderer to create a copy of the component.